### DISCUSSION TITLE
MCU2 bluetooth crashing when pairing with Tesla Android

### DISCUSSION THREAD

**Discussion by RetoHB** [tesla-android/issue-tracker#261] [Q&A] [answered]
For better audio performance I wanted to pair my Tesla Android (2023.18.1) with the car's bluetooth (MCU2). But when I try to pair the devices the car bluetooth seems to crash. The connection to my phone disappears from the car screen and the only button left is "add new phone". But this button is unresponsive. I cannot use bluetooth anymore until a reset or deepsleep of the car.

**Comment by mikegapinski** [comment]
What firmware version is your car on? It works fine on my two MCU2 devices and most of my testers don't have amd also...

**Comment by RetoHB** [comment]
2023.12.1 but I had it on the version before as well.

**Reply by mikegapinski** [reply, marked as answer]
One tip. Try pairing another BT device to your Pi like AirPods. If it does not work try updating the boot firmware(eeprom) on the Pi. It helped some people on Reddit with Wi-Fi issues

The drivers for BT/Wi-Fi were updated in this version, if your boot firmware is very old it might interfere

**Reply by RetoHB** [reply]
Thanks!! Updating the bootloader fixed it!
